src/commonTest/kotlin/fr/acinq/lightning/transactions/AnchorOutputsTestsCommon.kt

```diff
@@ -90,7 +88,8 @@ class AnchorOutputsTestsCommon {
     private fun runHighLevelTest(testCase: TestCase) {
         val localParams = LocalParams(
             TestConstants.Alice.nodeParams.nodeId,
-            KeyPath.empty, 546.sat, 1000000000L, 0.sat, 0.msat, CltvExpiryDelta(144), 1000, true,
+            ChannelKeys( KeyPath.empty, local_funding_privkey, local_payment_basepoint_secret, local_delayed_payment_basepoint_secret, local_payment_basepoint_secret, local_payment_basepoint_secret, randomBytes32()),
```

Comment:
```suggestion
            ChannelKeys(KeyPath.empty, local_funding_privkey, local_payment_basepoint_secret, local_delayed_payment_basepoint_secret, local_payment_basepoint_secret, local_payment_basepoint_secret, randomBytes32()),
```

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/255/commits/3b01af57bbb8bc95af625ce1d11ae54e433214de